### PR TITLE
Cosmos Spark: Update quickstart

### PR DESCRIPTION
**Pull request by ealsur**

### CODE REVIEW

File: sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/quick-start.md [hunk 1]
```diff
@@ -21,22 +17,16 @@ You can use any other Spark 3.1.1 spark offering as well, also you should be abl
...
 
-SLF4J is only needed if you plan to use logging, please also download an SLF4J binding which will
-link the SLF4J API with the logging implementation of your choice. See
-the [SLF4J user manual](https://www.slf4j.org/manual.html) for more information.
-
-You need a Spark 3.0.1 Environment.
+SLF4J is only needed if you plan to use logging, please also download an SLF4J binding which will link the SLF4J API with the logging implementation of your choice. See the [SLF4J user manual](https://www.slf4j.org/manual.html) for more information.
 
-Install Cosmos DB Spark Connector, in your spark Cluster `azure-cosmos-spark_3-1_2-12-4.0.0-beta.1.jar`
+Install Cosmos DB Spark Connector, in your spark Cluster [azure-cosmos-spark_3-1_2-12-4.1.0.jar](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-1_2-12/4.1.0/jar)
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/quick-start.md:22]: I have changed the reference to 4.1.0

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM - only changed the reference to version 4.1.0

File: sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/quick-start.md (hunk 1 above)
Comment (ealsur) [reply sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/quick-start.md:22]: Thanks!

**Review by moderakh — APPROVED** [review]
LGTM.

**Pull request merged**